get-involved.md

```diff
@@ -20,7 +20,7 @@ or with your IRC client:
 
 ### IRC Logs
 
-The IRC logs are hosted at: [/irclogs/](/irclogs/)
+The IRC logs are hosted at: [/irclogs/](http://pelux.io/irclogs/)
```

Comment:
If we're putting internal links with the scheme then they should be consistent, either http or https, but I'd rather see only absolute links like it was before if that's possible. Because then changing between http and https would work automatically, and even changing the domain wouldn't need any changes to the links.

Reply:
I agree and this would be nice. The problem is that when building this in the CI, the /irclogs/ path points nowhere, since that path is served by another process on another server.

I'll update so that all links are https wherever possible.

Reply:
You could also write `[/irclogs/](//pelux.io/irclogs/)` then it would use http or https depending on from which one it was served.

Reply:
Will that work with any scheme? I'll try it out.

Reply:
Yes it should.

Reply:
I noticed this will make those links exempt from the htmlproofing though, but perhaps we can live with that since we have other measures to check that our own things are working?

Reply:
Oh, interesting, I would almost say that this is a bug in the htmlproofing, but yeah for now I guess.